interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: 'this' is a continuation of the function so that it can be called upon later in the code
 
-  Researched answer:
+  Researched answer: This is to be able to reference the object that you are using within the Class.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React allows the user to be able to see their code in real time and also allows the user to interact with the code in real time without having to reload the page.
 
-  Researched answer:
+  Researched answer: React is a javascript library that enhances someones code by making it interactice and easy to use UI
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: This is something that i do no recall but I will research the answer and get it to you as soon as I can. 
 
-  Researched answer:
+  Researched answer: This is the method that is called when the component is mounted to the DOM.
```

Comment:
The method is the render() method as React always needs to render something to the page.